### ISSUE TITLE
Specify Time Range In Agenda

### ISSUE THREAD

**Issue by parhamdoustdar** [insanum/gcalcli#430]
I'm using gcalcli version 4.0.3, and I was wondering if it's possible to provide the time range in the agenda output. Here is an example of a command I'd run:

```
$ gcalcli --calendar my-calendar --nocolor agenda > ~/.emacs.d/diary
```

Here is the output I expect to see:

```
Mon Mar 11  11:00am-11:15am  Lifecycle Stand-up
```

What I get is:

```
Mon Mar 11  11:00am  Lifecycle Stand-up
```

I can add the `--details length` to the command, but then that gives me something like `00:15:00`, not something like `11:30am-12:10pm`.

BTW, this application is amazing. Thank you for making my life easier!

**Comment by jcrowgey** [comment]
If I understand you correctly, you'd like to see the "stop time" as well as the start time.  I'm not opposed to such a feature.  In the meantime, as a workaround, you can add `--details=length` to your invocation.

```shell
$ gcalcli  --calendar=testcal@gmail.com add
Title: a test event
Location: somewhere
When: 9am
Duration (minutes): 75
Description: a test
Enter a valid reminder or "." to end: .
$ gcalcli agenda --details=length

Sat Mar 09   9:00am  a test event
                     Length: 1:15:00

Sun Mar 10           Daylight Saving Time starts
                     Length: 23:00:00
```

**Comment by parhamdoustdar** [comment]
Yes! Thanks a lot for mentioning this – it's what I do now. However, I'm trying to use the output with the agenda in Emacs and it understands time ranges (i.e. `<start-time>-<end-time>`). As it stands right now, I have to open the description of each event in the agenda to view the `length`.

**Comment by dirtbag** [comment]
Id also like to see this ability to see the start AND stop time as mentioned above when I run 
gcalcli calw.
